app/views/root/_javascript.html.erb

```diff
@@ -1,3 +1,6 @@
 <%= javascript_include_tag 'libs/jquery/jquery-1.12.4.js', integrity: true, crossorigin: 'anonymous' %>
 <%= javascript_include_tag local_assigns[:js_file] || 'application', integrity: true, crossorigin: 'anonymous' %>
+<% if ENV["DRAFT_ENVIRONMENT"].present? %>
+  <%= javascript_include_tag 'modules/base-target', integrity: true, crossorigin: 'anonymous' %>
+<% end %>
```

Comment:
I'd have thought we'd want this to be a general draft script that includes this module rather than including this module specifically.

Reply:
I would not generalise this until we have at least two modules running only in the draft environment.